src/sentry/migrations/0292_migrate_sessions_subs_user_counts.py

```diff
@@ -2,22 +2,51 @@
 # Based on https://github.com/getsentry/getsentry/blob/89ff1453be755ddef31f2b99de09bd03badeb25e/getsentry/migrations/0141_migrate_sessions_subs_to_metrics.py
 
 import logging
+import re
 
 from django.db import migrations
 
 from sentry.new_migrations.migrations import CheckedMigration
 from sentry.snuba.dataset import EntityKey
-from sentry.snuba.entity_subscription import map_aggregate_to_entity_key
 from sentry.snuba.models import QueryDatasets
 from sentry.snuba.tasks import _create_in_snuba, _delete_from_snuba
 from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
 
+CRASH_RATE_ALERT_AGGREGATE_RE = (
+    r"^percentage\([ ]*(sessions_crashed|users_crashed)[ ]*\,[ ]*(sessions|users)[ ]*\)"
+)
+
 
 def create_subscription_in_snuba(subscription):
     subscription.subscription_id = _create_in_snuba(subscription)
     subscription.save()
 
 
+def map_aggregate_to_entity_key(dataset: QueryDatasets, aggregate: str) -> EntityKey:
```

Comment:
I moved this logic into the migrations because we usually don't want to rely on external logic when possible, and I'm deleting the external version of this function.